src/common/IterationLookup.ts

```diff
@@ -265,21 +283,23 @@ export class Iterations implements IterationLookup {
    * (or today if no date is given).
    * @param {number} diff n for upcoming iterations, -n for previous iterations
    * @param {string|DateTime} [dateString] defaults to today
-   * @returns {LegacyIteration}
+   * @returns {LegacyIteration|null}
    */
   getAdjacentIteration(
     diff: number,
     dateString?: string | DateTime
   ): LegacyIteration {
```

Comment:
actually, you need to change the return types in the signature here too. it should say
```suggestion
  ): LegacyIteration | null {
```